articles/django-app-using-postgresql-database/index.md

```diff
@@ -0,0 +1,211 @@
+    'hello_world',
+]
+```
+
+### Step 3 - set up your project to use PostgreSQL Database 
+navigate to django_app/settings.py
+go to the DATABASE SECTION 
+```python 
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
+```
+change the above code to this
+
+```python 
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'helloworld',
+        'USER': '<yourname>',
+        'PASSWORD': 'password',
+        'HOST': 'localhost',
+        'PORT': '',
+    }
+}
+```
+we changed the backend engine to use postgresql_psycopg2 instead of the default sqlite3.
+Where- NAME is the name of the database we created for our project,
+     - USER is the database user we created during the database creation,
+     - PASSWORD is the password to the database we created,
+
+Now let make sure our django project is working without errors.
+ 
+```python
+python manage.py makemigrations
+python manage.py migrate
+```
+after running the above commands you will get something like this in your CLI
+```bash 
+Operations to perform:
+  Apply all migrations: admin, auth, contenttypes, sessions
+Running migrations:
+  Applying contenttypes.0001_initial... OK
+  Applying auth.0001_initial... OK
+  Applying admin.0001_initial... OK
+  ....
+  ....
+  Applying auth.0012_alter_user_first_name_max_length... OK
+  Applying sessions.0001_initial... OK
+```
+To verify our django app is working run this command: 
+```bash
+python manage.py runserver
+```
+after running the above commands proceed to visit 127.0.0.1:8000
+if your app runs without errors you should get the below picture. 
+![Django Homepage](/engineering-education/django-app-using-postgresql-database/django_webpage.jpg)
+Congratulations you configured Django to use a PostgresSQL.
+
+### Hello World in Django.
+
+First let create a templates folder in our django_app/templates directory and create a index.html file. Your project directory should look like this.
+```
+--django_app
+| ---django_app
+| ---hello_world
+| ---templates
+|     --- index.html
+| ---manage.py
+
+```
+Let configure our app to make use of the templates folder. Head over to django_app/settings.py the TEMPLATES section.
+```python 
+ TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            Path(BASE_DIR, 'templates') #include this line to your file
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
+```
+To make sure our django templates is now working let go me work you through this short steps
```

Comment:
Rewrite this line. `let go me work you through this short steps`. This part doesn't make sense.